docs/releases/v8_4.md

```diff
@@ -0,0 +1,48 @@
+# Release Notes
+
+## April 2022, vol 2 (version 8.4)
+
+### Overview
+
+The **April 30th, 2022** release of **DietPi v8.4** comes together with new images for NanoPi and ODROID, enhanced `dietpi-software` usability, and more.
+
+![daisy flowers](../assets/images/dietpi-release-v8_4.jpg){: width="426" height="283" loading="lazy"}
+
+!!! cite "_Photo by Anestiev, Pixabay_"
+
+### New software
+
+- None.
```

Comment:
Do you want to keep this, even if the heading has no content? I'd simply remove it and only list headings/blocks were something really has changed.